### PR TITLE
♻️ Refactor underscore usage

### LINKED ISSUE #78: Leading Underscores in Variable Names
https://github.com/arcadia-finance/lending-core/blob/6eb1f213245765cd7d7a0f8b562fecd8669a2c9e/src/Vault.sol#L40-L61

Why are some variables named with leading underscores (eg. Vault.sol, line 40 - 53)  while others are not (Vault.sol, line 57 - 61)?

### PR DESCRIPTION
**Pull request by alikonuk1**
This pr refactored underscore usage and consistency through the codebase. 
- Preceding underscore for internal functions
- Following underscore for variables when there is a clash

Closes #78

### CODE REVIEW

File: src/Liquidator.sol [hunk 1]
```diff
@@ -71,28 +71,28 @@ contract Liquidator is Ownable {
...
     /**
      * @notice Sets the protocol treasury address on the liquidator.
      * @dev The protocol treasury is used to receive liquidation rewards.
-     * @param _protocolTreasury the protocol treasury.
+     * @param protocolTreasury_ the protocol treasury.
      */
-    function setProtocolTreasury(address _protocolTreasury) external onlyOwner {
-        protocolTreasury = _protocolTreasury;
+    function setProtocolTreasury(address protocolTreasury_) external onlyOwner {
+        protocolTreasury = protocolTreasury_;
```
Comment (zekiblue) [inline src/Liquidator.sol:86]: maybe to refactor this to 
```suggestion
        protocolTreasuryAddress = protocolTreasury;
```
what do you think @Thomas-Smets @j-vp

Comment (zekiblue) [reply src/Liquidator.sol:86]: or change `factoryAddress` to `factory` in the contract for consistent naming

File: src/Liquidator.sol [hunk 2]
```diff
@@ -71,28 +71,28 @@ contract Liquidator is Ownable {
...
     }
 
     /**
      * @notice Sets the reserve fund address on the liquidator.
      * @dev The reserve fund is used to pay liquidation keepers should the liquidation surplus be insufficient.
-     * @param _reserveFund the reserve fund address.
+     * @param reserveFund_ the reserve fund address.
      */
-    function setReserveFund(address _reserveFund) external onlyOwner {
-        reserveFund = _reserveFund;
```
Comment (zekiblue) [inline src/Liquidator.sol:95]: same goes here as i pointed, in the `protocolTreasury` line

**Review by zekiblue — CHANGES_REQUESTED** [review]

File: src/Liquidator.sol (hunk 1 above)
Comment (Thomas-Smets) [reply src/Liquidator.sol:86]: Jep, the shorter the name the better.
Also bad practice to repeat the type in the name

File: src/Liquidator.sol [hunk 3]
```diff
@@ -71,28 +71,28 @@ contract Liquidator is Ownable {
     /**
      * @notice Sets the factory address on the liquidator.
      * @dev The factory is used to fetch the isVault bool in elevated().
-     * @param _factory the factory address.
+     * @param factory the factory address.
      */
-    function setFactory(address _factory) external onlyOwner {
-        factoryAddress = _factory;
+    function setFactory(address factory) external onlyOwner {
+        factoryAddress = factory;
```
Comment (j-vp) [inline src/Liquidator.sol:77]:
```suggestion
        factory = factory_;
```

File: src/Liquidator.sol [hunk 4]
```diff
@@ -71,28 +71,28 @@ contract Liquidator is Ownable {
     /**
      * @notice Sets the factory address on the liquidator.
      * @dev The factory is used to fetch the isVault bool in elevated().
-     * @param _factory the factory address.
+     * @param factory the factory address.
      */
-    function setFactory(address _factory) external onlyOwner {
-        factoryAddress = _factory;
+    function setFactory(address factory) external onlyOwner {
```
Comment (j-vp) [inline src/Liquidator.sol:76]:
```suggestion
    function setFactory(address factory_) external onlyOwner {
```
(also update the storage variable name)

**Review by j-vp — COMMENTED** [review]
@alikonuk1 I think there are also some shadowing variables in the constructors of the pricing modules

File: src/interfaces/IOraclesHub.sol [hunk 5]
```diff
@@ -9,5 +9,5 @@ pragma solidity >=0.4.22 <0.9.0;
 interface IOraclesHub {
     function getRate(address[] memory, uint256) external view returns (uint256, uint256);
 
-    function checkOracleSequence(address[] memory oracleAdresses) external view;
+    function checkOracleSequence(address[] memory oracle) external view;
```
Comment (Thomas-Smets) [inline src/interfaces/IOraclesHub.sol:12]:
```suggestion
    function checkOracleSequence(address[] memory oracles) external view;
```

**Review by Thomas-Smets — APPROVED** [review]

**Review by zekiblue — APPROVED** [review]
gtg

**Pull request merged**